feature/system/management_vrf/README.md

```diff
@@ -0,0 +1,85 @@
+
+## Test input args
+
+*   `mgmt_vrf_name`: name of the management VRF the test should use. If the lab infrastructure is already using a non default VRF to manage the device, we recommend to specify the name here so that the test reuses the VRF to avoid unnecessary interruptions.
+*   `lab_infra_loopback_name`: before the test starts, this is the name of the loopback interface that hosts the IPv6 management IP. During the test, we will extract the IPv6 management IP from the specified interface, and  put it under a loopback subinterface 100 (say lo0.100) in the management VRF.
+*   `inband_interfaces`: before the tests start, this is a list of linecard interfaces that the lab infrastructure uses to manage the device. During the test, we will ensure it’s in the management VRF.
+
+## Baseline setup
+
+*   It is strongly recommended to do the following configs in one gNMI SetRequest to minimize the chances of getting the device in an isolated/unmanageable state. **Note**: the management connection might disconnect in this step. However, reconnection should happen successfully within 3min. **Note**: though OC is preferred, it’s ok to configure the following in vendor CLI:
+    *   Configure the following interfaces are in the `mgmt_vrf_name`VRF:
+        *   lo0.100, which is configured with only the IPv6 management IP that’s extracted from `lab_infra_loopback_name`. No IPv4 address is configured under lo0.100.
+        *   Port-1, port-2, port-11, port-12 and OOB interfaces.
+    *   Configure the following services to listen on the lo0.100 management IP, with [IANA assigned ports](https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml). If a service has no assigned port yet, it should be provided via a test input arguments.
+        *   gRIBI
+        *   gNOI
+        *   gNMI
+        *   P4RT
+    *   Configure the following services with the expectation that lo0.100 IP will be used for communication.
+        *   sFLOW
+        *   Syslog
+        *   TACACS services.
+        *   SSH services.
+        *   Specify the target name server.
+        *   Specify the target NTP server.
+    *   Configure two `::/0` static routes configured in the management VRF, with default route preference and next hops being the adjacent IP of port-11 and port-12. This will be the primary channel for management traffic. In prod, this will be BGP routes. However, we don’t have to enforce this on the lab infrastructure.
+    *   Configure a `::/0` static route configured in the management VRF with route preference 220 going via OOB interface. This will be the backup channel for management traffic.
+*   Configure a v6 eBGP between DUT port-1 and ATE port-1.
+*   Configure a v6 eBGP between DUT port-2 and ATE port-2.
+*   Configure ATE to advertise a prefix (say, [2001:DB8:0::/64](https://github.com/openconfig/featureprofiles/blob/main/CONTRIBUTING.md#ipv6)) to DUT.
```

Comment:
Just idea ... Perhaps we can use foregin-next-hop? Not sure it will work...
```suggestion
*   Configure ATE port-1 to advertise a ::/0 prefix to DUT with BGP next-hop attribute equalt to adjacent address of **port-11**.
*   Configure ATE port-2 to advertise a ::/0 prefix to DUT with BGP next-hop attribute equalt to adjacent address of **port-12**.
* delete any static routing with next-hop of adjacent IP of Port-1 and port-2
```
This way, we trick system to have ::/0 BGP routes pointing "right" exit ports !